### DISCUSSION TITLE
表格按钮事件点击 如何获取点击按钮elem

### DISCUSSION THREAD

**Discussion by ken678** [layui/layui#2079] [讨论]
![image](https://github.com/layui/layui/assets/15792472/cec3dc0d-49b4-4ee0-b86c-ce21b6fdaba5)

**Comment by ken678** [comment]
看文档没有获取按钮elem，只能这么获取$('[lay-event="add"]') ？？

**Comment by Sight-wcg** [comment]
表格事件处理程序中，this 指向当前触发事件的 DOM 元素。

**Reply by ken678** [reply]
一开始我知道this 测试不行 大神你又回复说了this   仔细检查了下我才发现this写到下面一行了 犯了低级错误  感谢感谢 自己没认真测试
